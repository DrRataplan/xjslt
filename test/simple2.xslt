<xsl:stylesheet
    version="1.0"
    xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

  <xsl:output method="xml" />

  <xsl:template match="/">
    <doc>
      <xsl:apply-templates/>
    </doc>
  </xsl:template>

  <xsl:template match="Article">
    <heading type="top"><xsl:value-of select="Title" /></heading>
    <list>
      <xsl:apply-templates select="Authors/Author" />
    </list>
  </xsl:template>

  <xsl:template match="Author">
    <item><xsl:value-of select="." /></item>
  </xsl:template>
</xsl:stylesheet>
