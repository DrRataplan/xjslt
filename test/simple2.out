<doc><heading type="top">My Article</heading><list><item>Mr. Foo</item><item>Mr. Bar</item></list></doc>